src/entities/dimension.ts

```diff
@@ -22,7 +19,7 @@ export class Dimension extends BaseEntity {
     @JoinColumn({ name: 'dataset_id', foreignKeyConstraintName: 'FK_dimension_dataset_id' })
     dataset: Promise<Dataset>;
 
-    @Column({ type: 'enum', enum: Object.keys(DimensionType), nullable: false })
+    @Column({ type: 'enum', enum: Object.values(DimensionType), nullable: false })
```

Comment:
Everywhere else we were using the enum values rather than enum keys so changed this for consistency.